scripts/components/BucketGroups.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { Link } from "react-router";
+
+import BucketTabs from "./BucketTabs";
+
+
+class DataList extends Component {
+  render() {
+    const {
+      bid,
+      data,
+      listLoaded,
+    } = this.props;
+    return (
+      <table className="table table-striped table-bordered record-list">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Members</th>
+            <th>Last mod.</th>
+          </tr>
+        </thead>
+        <tbody className={!listLoaded ? "loading" : ""}>{
+          data.map((group, index) => {
```

Comment:
Same remark as for destructuring object properties, see previous remark about it.